src/OVAL/oval_probe.c

```diff
@@ -514,7 +514,11 @@ void oval_probe_meta_list(FILE *output, int flags)
 
 			if (flags & OVAL_PROBEMETA_LIST_DYNAMIC) {
 				dI("Checking access to \"%s\"", probe_path);
+#ifdef _WIN32
```

Comment:
What about having an `#ifdef` section at the top of the file and declare `X_OK` to zero?

Reply:
Thank you, I agree, that is definitely better.